app/src/main/res/values/strings.xml

```diff
@@ -884,7 +884,7 @@
     <string name="type_uri">URI</string>
     <string name="no_battery_optimization">No battery optimization</string>
     <string name="battery_optimization">Battery optimization</string>
-    <string name="enable_battery_optimization">Enable battery optimization?</string>
+    <string name="enable_battery_optimization">Enable battery optimization for AppManager?</string>
```

Comment:
This is not for App Manager.

Reply:
What is it for? Some translations differ between options.

Reply:
For any app.

Reply:
Is it something like "Let any app use battery optimization if requested"?
I don't understand the context, and I can't find it in the app.

Reply:
No, it does exactly what the settings do. You can find it under the three dots menu in the app info tab in v2.5.23 or later provided you have root.

Reply:
I do, but I'll have to get something more recent than F-Droid's version. Not changed in this PR though, so for another day to try and find possible improvements there.